app/templates/settings/labs.hbs

```diff
@@ -179,65 +179,93 @@
 
-                                        <div class="form-group">
-                                            <label for="oauth-client-secret">Google OAuth Client Secret</label>
-                                            <GhTextInput
-                                                id="oauth-client-secret"
-                                                @value={{readonly this.settings.oauthClientSecret}}
-                                                @name="oauth-client-secret"
-                                                @focus-out={{action "saveOAuthSettings"}}
-                                                @input={{action (mut this.settings.oauthClientSecret) value="target.value"}}
-                                            />
-                                        </div>
-                                    </GhFormGroup>
-                                </div>
-                            {{/if}}
+                                    <div class="form-group">
+                                        <label for="oauth-client-secret">Google OAuth Client Secret</label>
+                                        <GhTextInput
+                                            id="oauth-client-secret"
+                                            @value={{readonly this.settings.oauthClientSecret}}
+                                            @name="oauth-client-secret"
+                                            @focus-out={{action "saveOAuthSettings"}}
+                                            @input={{action (mut this.settings.oauthClientSecret) value="target.value"}}
+                                        />
+                                    </div>
+                                </GhFormGroup>
+                            </div>
+                        {{/if}}
+                    </div>
+                </div>
+                <div class="gh-expandable-block">
+                    <div class="gh-expandable-header">
+                        <div>
+                            <h4 class="gh-expandable-title">Match helper</h4>
+                            <p class="gh-expandable-description">
+                               Experimental <code>\{{match}}</code> helper for themes.
+                            </p>
+                        </div>
+                        <div class="for-switch">
+                            <label class="switch" for="settings-labs-match">
+                                <input
+                                    type="checkbox"
+                                    checked={{this.isMatchEnabled}}
+                                    id="settings-labs-match"
+                                    onclick={{action "toggleLabsMatchHelper" value="target.checked"}}
+                                    data-test-private-checkbox
+                                >
+                                <span class="input-toggle-component"></span>
+                            </label>
```

Comment:
I think this can be replaced with `<GhFeatureFlag @flag="matchHelper" />` then the `isMatchEnabled` and `toggleLabsMatchHelper` changes on the controller can be removed.

The only change that is needed I think is to change `checkbox` to `switch` here https://github.com/TryGhost/Admin/blob/main/app/components/gh-feature-flag.js#L10

Reply:
Oooooooh

Reply:
Yeah that totally works 👍